README.md

```diff
@@ -1,131 +1,335 @@
+
+Options are disabled by default. Adding an option to `project_options` will enable.
 
+```sh
+--docs  "Add docs in project vendor directory"
+
+--local  "Create a copy of Ceedling in the project vendor directory"
+
+--gitignore  "Create a gitignore file for ignoring ceedling generated files"
+
+--no_configs  "Don't install starter configuration files"
+```
 
 #### Adding Ceedling to existing projects
-The `New Project` module can also be used to add Ceedling support to exisiting source code.\
+The `New Project` module can be used to add Ceedling support to exisiting source code.\
 Existing files and folders are not overwritten or modified in the process.\
 Use the path of the existing project when prompted for the location.
 
 
 ### Create New Module
+
 The `Create New Module` command now uses paths relative to the project folder.
+
 Specifying `basename` will generate the files `test/test_basename.c`, `src/basename.h` and `src/basename.c` by default.
 
 Additional naming schemes are supported using an abbreviated format identifiers:
 
-* mch - model, conductor, hardware
-* mvp - model, view, presenter
-* dhi - driver, hardware, interrupt
-* dh - driver, hardware
-* test - test file only
-* src - header, source, test (default output)
+| Abbreviation | Naming Scheme | Files |
+|:--|:--|--:|
+| src | header, source, test (default) | 3 |
+| test | test file only | 1 |
+| mch | model, conductor, hardware | 9 |
+| mvp | model, view, presenter | 9 |
+| dhi | driver, hardware, interrupt | 9 |
+| dh | driver, hardware | 6 |
 
-The first four schemes generate header, source and test files for each of the named variants.
+`mch`, `mvp`, `dhi` and `dh` schemes generate header, source and test files for each of the named modules.
 
-## Running tests
-Sublime Text build system is used to run tests.
+### Running tests
+The Sublime Text build system is used to run all tests.
 
-From `Tools >> Build System` menu select `Ceedling` as the build system for the project.
+From `Tools » Build System` menu select `Ceedling` as the build system for the project.
 
 | Variant | Ceedling Task | Notes |
 |:--|:--|:--|
 | Default | `test:filename` | Test current module|
-| Test All | `test:all` | |
-| Test Changes | `test:delta` | |
-| Test Build only | `test:build_only`  | |
+| Test All | `test:all` | Test all modules|
+| Test Changes | `test:delta` | Test changed modules |
+| Test Build only | `test:build_only`  | Build all without testing |
 | Clean and Test file | `clean test:filename` | |
 | Release | `release` | The release config is disabled in new project.yml. |
 
-Select variants using `ctrl-shift-b`/`command-shift-b`.
+Select variants using `Control-Shift-B` (Windows, Linux) /`Command-Shift-B` (macOS).
+
+Run last selected build variant using `Control-B` (Windows, Linux) / `Command-B`(macOS).
 
-Run selected build using `ctrl-b` / `command-b`.
 
+### Key mappings
 
-## Key commands
+A number of key commands for working with modules are predefined.
```

Comment:
It's not exactly obvious what this means to me. I think you meant to suggest the following key bindings but the wording hints more at these already being defined.

Reply:
The text has been altered to reflect that key mapping is commented out and can be copied to user Default key map if desired.